todo_app/test/test_integration.py

```diff
@@ -0,0 +1,39 @@
+import pytest, requests, os
+from dotenv import load_dotenv, find_dotenv
+from todo_app import app
+
+@pytest.fixture 
+def client():
+    # Use our test integration config instead of the 'real' version 
+    file_path = find_dotenv('.env.test')
+    load_dotenv(file_path, override=True)
+
+    # Create the new app. 
+    test_app = app.create_app()
+
+    # Use the app to create a test_client that can be used in our tests.
+    with test_app.test_client() as client:
+        yield client
+
+def test_index_page(monkeypatch, client):
+    # Replace call to requests.get(url) with our own function
+    monkeypatch.setattr(requests, 'get', get_lists_stub)
+    response = client.get('/')
+    assert response.status_code == 200
```

Comment:
Although this is a good start, it's worth us adding an additional assertion to try and determine whether the fake data is actually being displayed correctly - getting back a 200 result but an empty page isn't much use! Something like the below would help, though see the comment because I believe this catches a genuine issue with the test as it currently stands

```suggestion
    assert response.status_code == 200
    assert "Test card" in response.data.decode()
```